README.md

```diff
@@ -35,10 +35,8 @@ directories. Please see the [LICENSE](LICENSE) file for more information.
 
 1. PowerShell 3.0 or above
 2. Net Framework 4.5 and 4.8 installed.
-3. Visual Studio build tools for 2019 (toolsversion 16.0 and toolset v142 for C++).

```

Comment:
We still need VS 2019 toolsversion 16.

Reply:
And the note about msbuild.

Reply:
So basically only the toolset v142 needed removal.

Reply:
Ah sorry I had misunderstood. Will fix it shortly